docs/sources/administration/plugin-management/index.md

```diff
@@ -162,21 +152,30 @@ plugins:
   - redis-datasource
 ```
 
-### Install plugin on local Grafana
+When the update is complete, you'll see a confirmation message that the installation was successful.
 
-Follow the instructions on the **Install** tab. You can either install the plugin with a Grafana CLI command or by downloading and uncompressing a zip file into the Grafana plugins directory. We recommend using Grafana CLI in most instances. The zip option is available if your Grafana server doesn't have access to the internet.
+### Update a plugin
 
-For more information about Grafana CLI plugin commands, refer to [Plugin commands]({{< relref "../../cli/#plugins-commands" >}}).
+To update a plugin:
 
-#### Install a packaged plugin
+1. In Grafana, click **Administration > Plugins and data > Plugins** in the side navigation menu to view all plugins.
+1. Click the **Installed** filter to show only installed plugins.
+1. Click the plugin's logo.
+1. Click **Update**.
 
-After the user has downloaded the archive containing the plugin assets, they can install it by extracting the archive into their plugin directory. For example:
+When the update is complete, you'll see a confirmation message that the update was successful.
```

Comment:
When the update is complete, a confirmation message will indicate it was successful.

Reply:
thanks, adjusted